tests/functional/spec/previous.spec.js

```diff
@@ -0,0 +1,42 @@
+const helpers = require('../helpers');
+const EmploymentStatusBlockPage = require('../generated_pages/hub_and_spoke/employment-status.page.js');
+const EmploymentTypePage = require('../generated_pages/hub_and_spoke/employment-type.page.js');
+const HubPage = require('../base_pages/hub.page.js');
+const ProxyPage = require('../generated_pages/hub_and_spoke/proxy.page.js');
+const schema = 'test_hub_complete_sections.json';
```

Comment:
Not sure if this test should be part of the /hub?. And it might be working checking that the previous link is as expected, ie they go where they should?.

Reply:
Moved it to /hub_and_spoke. Thanks!